packages/financial-templates-lib/test/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -212,8 +212,9 @@ contract("CryptoWatchPriceFeed.js", function(accounts) {
       }
     ];
 
-    await cryptoWatchPriceFeed.update();
-    await invertedCryptoWatchPriceFeed.update();
+    // Update should throw errors in both cases.
+    assert.isTrue(await cryptoWatchPriceFeed.update().catch(() => true), "Update didn't throw");
```

Comment:
tiny nit: why are you throwing an error message in the `assert.isTrue`? I like this syntax but I don't think we do this anywhere else to show the error reason. you can leave it, but just wanted to point out that it's different to other tests.

Reply:
I added the error message because it's pretty hard to tell what's being tested due to the syntax. I feel like assert.equal(X,Y) is pretty obvious, but a `await cryptoWatchPriceFeed.update().catch(() => true)` is a very opaque way to tell if an async method is throwing (although it's the only concise way I could find). Happy to change to a non error message or just add a comment if you think that would be more readable/consistent!

Reply:
i wonder if you can use assert.throw https://stackoverflow.com/questions/14966821/testing-for-errors-thrown-in-mocha
maybe not though if you dont know exact error

Reply:
`assert.throws()` is what I tried first, but I don't think it handles async functions :/

Reply:
i see yea that makes sense then. that is an really clever work around